library/julia

```diff
@@ -1,5 +1,6 @@
 # maintainer: InfoSiftr <github@infosiftr.com> (@infosiftr)
 
+0.4: git://github.com/docker-library/julia@433ee54173ff0b8900045f5a81cad96b34558e9f
```

Comment:
Yeah, this whole file needs to be generated using the
"generate-stackbrew-library.sh" script in the julia repository.